build-recipe-openwrt

```diff
@@ -0,0 +1,128 @@
+#################################################################
+#
+# OpenWrt specific functions.
+#
+# Adapted from Flatpak specific functions, see details below.
+#
+# Author: Tina Müller <tina.mueller@suse.com>
```

Comment:
You might want to change this part, at least the author is incorrect ;-)

Reply:
Thanks, let me know if this works.